azurelinuxagent/common/protocol/wire.py

```diff
@@ -726,7 +759,7 @@ def update_goal_state(self, forced=False, max_retry=3):
                                         last_incarnation == new_incarnation:
                             # Goalstate is not updated.
                             return
-
+                
```

Comment:
NIT: this is an unnecessary whitespace change

Reply:
done